src/lib/tooltip/tooltip.ts

```diff
@@ -255,7 +255,7 @@ export class MatTooltip implements OnDestroy {
     }
 
     // Clean up the event listeners set in the constructor
-    if (!this._platform.IOS) {
+    if (!this._platform.IOS && !this._platform.ANDROID) {
```

Comment:
The `if` statement around this call can be removed since nothing will be set to the `_manualListeners` if it's not iOS or Android and the loop won't do anything.